cmd/liqoctl/cmd/install.go

```diff
@@ -174,6 +181,8 @@ func newInstallCommand(ctx context.Context, f *factory.Factory) *cobra.Command {
 	cmd.PersistentFlags().BoolVar(&options.EnableMetrics, "enable-metrics", false, "Enable metrics exposition through prometheus (default false)")
 	cmd.PersistentFlags().BoolVar(&options.DisableTelemetry, "disable-telemetry", false,
 		"Disable the anonymous and aggregated Liqo telemetry collection (default false)")
+	cmd.PersistentFlags().BoolVar(&options.EnableKernelVersionCheck, "enable-kernel-version-check", true,
```

Comment:
```suggestion
	cmd.PersistentFlags().BoolVar(&options.EnableKernelVersionCheck, "disable-kernel-version-check", false,
```

Disabling default true flags is a mess (you need to specify --flaname=false), while default false flags are easier to enable (you can specify --flagname only)